src/Mobs/Path.cpp

```diff
@@ -51,6 +51,14 @@ cPath::cPath(
 	m_Source.x = FloorC(a_StartingPoint.x - HalfWidthInt);
 	m_Source.y = FloorC(a_StartingPoint.y);
 	m_Source.z = FloorC(a_StartingPoint.z - HalfWidthInt);
+	
+	if ((m_Source.y <= 0) || (m_Destination.y <= 0)
+		|| (m_Source.y > 256) || (m_Destination.y > 256))
```

Comment:
Its should not problematic for a mob to sit on top of the world. The entities should be able to be outside the world. Players can certainly pass over the top of such blocks, and the code for mobs and players are the same in this instance.